internal/pkg/docker/userinfo_test.go

```diff
@@ -1,29 +1,16 @@
-package up
+package docker
```

Comment:
It's also a good idea we look at testing with importing the package. Perhaps a TODO for a later discussion around testing

Reply:
What do you mean?

Reply:
https://segment.com/blog/5-advanced-testing-techniques-in-go/

Reply:
> Use a separate _test package

Reply:
How do we test internal functions (lowercase first letter) if we should use a separate `_test` package?